types/hooks/screens/useMessageHook.types.ts

```diff
@@ -32,6 +33,7 @@ export interface UseMessageHook {
     data: any,
     type: 'ANONYMOUS' | 'SIGNED'
   ) => void;
+  onOpenMediaPreview: (medias: any, index: number, navigation: any) => void;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the `medias` parameter in the `onOpenMediaPreview` function is typed as `any`. It's recommended to replace this with a more specific type to avoid potential issues at runtime.

```diff
- onOpenMediaPreview: (medias: any, index: number, navigation: any) => void;
+ onOpenMediaPreview: (medias: Media[], index: number, navigation: NavigationType) => void;
```

Where `Media` and `NavigationType` are defined interfaces or types.


<!-- This is an auto-generated comment by OSS CodeRabbit -->